### ISSUE TITLE
Sentry 8.3.0 w/ MySQL 5.5 keep getting redirected to setup dialog after installation

### ISSUE THREAD

**Issue by ye** [getsentry/sentry#3019]
Clicking on "Overview" on the sidebar (/manage), it keeps redirecting me to ![setup-dialog](https://cloud.githubusercontent.com/assets/133924/14360899/745d06b4-fcc6-11e5-884f-cc99306a6140.png).

Btw, I have 3 Deprecated Settings:

- The EMAIL_BACKEND setting is deprecated. Please use SENTRY_OPTIONS['mail.backend'] instead.
- The SECRET_KEY setting is deprecated. Please use SENTRY_OPTIONS['system.secret-key'] instead.
- The SERVER_EMAIL setting is deprecated. Please use SENTRY_OPTIONS['mail.from'] instead.

Also, testing the API endpoint did not work even though the httpie CLI gets a 200 OK.

```bash
$ http -v -j  POST http://10.10.10.10:8080/api/2/store/ X-Sentry-Auth:"Sentry sentry_version=7,sentry_client=raven-lua/0.40,sentry_timestamp=2016-04-07T16:58:02Z,sentry_key=9576c37f61244c9ab0f48e698f6f8508,sentry_secret=2245c47657a44d0a9d71a2c9756cd86f" < test.json 
POST /api/2/store/ HTTP/1.1
Accept: application/json
Accept-Encoding: gzip, deflate
Connection: keep-alive
Content-Length: 315
Content-Type: application/json
Host: 10.10.10.10:8080
User-Agent: HTTPie/0.9.3
X-Sentry-Auth: Sentry sentry_version=7,sentry_client=raven-lua/0.40,sentry_timestamp=2016-04-07T16:58:02Z,sentry_key=9576c37f61244c9ab0f48e698f6f8508,sentry_secret=2245c47657a44d0a9d71a2c9756cd86f

{
    "culprit": "my.module.function_name", 
    "event_id": "fc6d8c0c43fc4630ad850ee518f1b9d0", 
    "exception": [
        {
            "module": "__builtins__", 
            "type": "SyntaxError", 
            "value": "Wattttt!"
        }
    ], 
    "message": "SyntaxError: Wattttt!", 
    "tags": {
        "ios_version": "4.0"
    }, 
    "timestamp": "2011-05-02T17:41:36"
}

HTTP/1.1 200 OK
Cache-Control: max-age=0
Content-Language: en
Content-Length: 41
Content-Type: application/json
Expires: Thu, 07 Apr 2016 17:02:04 GMT
Last-Modified: Thu, 07 Apr 2016 17:02:04 GMT
Vary: Accept-Language, Cookie

{
    "id": "fc6d8c0c43fc4630ad850ee518f1b9d0"
}
```

Moving some of my comments from #1441 to this as a new ticket.

**Comment by mattrobenolt** [comment]
What version of Sentry? There was a bug regarding this that got patched in 8.3.1.

**Comment by mattrobenolt** [comment]
Oh, you said 8.3.0. Sorry, confusing since the image is the screen shot from our blog that says 8.0.0. But either way, should be fixed with 8.3.1, so closing this.

**Comment by ye** [comment]
`8.3.0`

**Comment by mattrobenolt** [comment]
For context, see https://github.com/getsentry/sentry/issues/2970 and https://github.com/getsentry/sentry/pull/2991

**Comment by ye** [comment]
Thanks for the quick response @mattrobenolt ! When I started working on the MySQL fork of your official docker receipt, it was still `8.3.0` and I didn't notice that I was lucky to get that bug. Again, appreciate the quick response!

**Comment by mattrobenolt** [comment]
Yeah, for the 8.3.0 tag, a patch was applied to work around it before a proper fix in 8.3.1, so you were probably missing https://github.com/getsentry/docker-sentry/commit/1b7489e5c5467eeab791975a44a0c79008a31e8f
